pkg/test/framework/components/namespace/namespace.go

```diff
@@ -22,6 +22,29 @@ import (
 	"istio.io/istio/pkg/test/framework/resource"
 )
 
+// Config contains configuration information about the namespace instance
+// including naming information, sidecar injection config, and other component-specific labels
+// note: each component that adds additional custom k8s labels for control should have a field here
+type Config struct {
+	Prefix        string
+	SidecarConfig SidecarConfig
```

Comment:
There should be nothing named `Sidecar` in the namespace package ... same as below.